### PR TITLE
Trial's Portfolio Project for Review

### PR DESCRIPTION
**Pull request by ab-noori**
### 🛠 Built With:
- CSS
- HTML
### 🔢 Features:
- Navigation bar
- Welcome section
- Projects section
- Contacts section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
+    <link rel="stylesheet" href="styles.css" />
+    <title>Personal Portfolio</title>
+  </head>
+  <body>
+    <nav id="navbar">
+      <a id="active" href="#welcome-section">About</a>
+      <a href="#projects">Work</a>
+      <a href="#contact">Contact</a>
+    </nav>
```
Comment (ab-noori) [inline index.html:16]: It would be semantically better to put the navBar inside the Header element.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+            <a href="https://codepen.io/freeCodeCamp/full/zNqgVx"
+                target="_blank"
+                class="project project-tile">
+
+                <img class="project-image"
+                src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg"
+                alt="project"/>
+
+            </a>
+        </div>
```
Comment (ab-noori) [inline index.html:34]: Please add a title for your Projects section using the H1 element, Semantically you can use the H1 element once per each section.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+            href="https://www.linkedin.com/in/rachid-ait-m-bark-a11946118/"
+            target="_blank"
+            class="btn contact-details"
+            ><i class="fab fa-facebook-square"></i> Rachid's linkedin </a >
+          <a
+            id="profile-link"
+            href="https://www.linkedin.com/in/abdul-ali-noori-384b85195/"
+            target="_blank"
+            class="btn contact-details"
+            ><i class="fab fa-github"></i> Abdul Ali's linkedin</a>
```
Comment (ab-noori) [inline index.html:114]: It would be better to put these links inside the "ul" element. it is the best practice.

**Comment by GedeonTS** [comment]
# INVALID REVIEW REQUEST

Hi @ab-noori , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that you did not review yourself in.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by ab-noori — COMMENTED** [review]
## Hi @ab-noori 👋 ,
Great job! you have done most of the part very well 👏
You still need to work on some issues to go to the next project, but you are almost there!
## Highlights
CSS best practice ✔️
## Required Changes ♻️
- Please make your README file clear and more readable as much as possible. take a look at [this](https://github.com/microverseinc/readme-templatel) template file!
- Please add [Linters](https://github.com/microverseinc/linters-config) to your project and configure it locally. It helps you get rid of CSS spaces and make your code more readable.

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @ab-noori , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Good communication with the student 
✅ ... Good use of GitHub inline comment

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**